platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteInputSelection.java

```diff
@@ -52,13 +54,9 @@ public T getValueOrNull() {
 	}
 
 	@Override
-	public Optional<T> getValue() {
+	public T getValueOrThrow() {
```

Comment:
Note that the behavior of this method changed:
- Previously, it only threw upon illegal input.
- Now, it throws upon illegal or blank input.

Reply:
yes, great